### DISCUSSION TITLE
does the server image have gpu acceleration?

### DISCUSSION THREAD

**Discussion by develperbayman** [Joshua-Riek/ubuntu-rockchip#377] [Q&A] [answered]
iv been loving and hating the orange pi 5 scene im trying to find a distro that has working onboard microphone and gpu acceleration and npu support im noticing some distros dont recognize the npu on boot so im wondering if the server image has gpu acceleration like the desktop version or does it have all the same errors as the desktop version?

**Comment by StuartIanNaylor** [comment, marked as answer]
I presume this is sort of universal for at least the ubuntu image
https://docs.radxa.com/en/general-tutorial/panfork
Likely though Mesa Panthor will be soon and the current blob with the CSF firmware works for OpenCL headless.
